Makefile

```diff
@@ -134,8 +134,11 @@ add-license: addlicense
 TEMPLATES_DIR := templates
 PROVIDER_TEMPLATES_DIR := $(TEMPLATES_DIR)/provider
 export PROVIDER_TEMPLATES_DIR
+CLUSTER_TEMPLATES_DIR := $(TEMPLATES_DIR)/cluster
 CHARTS_PACKAGE_DIR ?= $(LOCALBIN)/charts
 EXTENSION_CHARTS_PACKAGE_DIR ?= $(LOCALBIN)/charts/extensions
+K0S_VERSION = $(shell $$YQ '.k0s.version' $(CLUSTER_TEMPLATES_DIR)/vsphere-standalone-cp/values.yaml)
```

Comment:
This has to be evaluated recursively because of `YQ` var (which is exported later).
Unfortunately this will be executed on each reference (4 times), but tricks with eval look worse IMO